unity/unity_memory_info.c

```diff
@@ -11,7 +11,7 @@
 #include <libgc/include/gc.h>
 #include <libgc/include/private/gc_priv.h>
 #include <mono/metadata/gc-internal.h>
-
+#include <mono/metadata/mono-debug-debugger.h>
```

Comment:
will probably need to check what happened to the definition of mono_vtable_get_static_field_data in mono 4.0 and if it needs the same include in here